sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -285,6 +292,38 @@ export interface KeyReleasePolicy {
     data?: Uint8Array;
 }
 
+// @public
+export interface KeyRotationLifetimeActions {
+    action?: "rotate" | "notify";
+    trigger?: KeyRotationLifetimeActionsTrigger;
+}
+
+// @public
+export interface KeyRotationLifetimeActionsTrigger {
+    timeAfterCreate?: string;
+    timeBeforeExpiry?: string;
+}
+
+// @public
+export interface KeyRotationPolicy {
+    attributes?: KeyRotationPolicyAttributes;
+    readonly id?: string;
+    lifetimeActions?: KeyRotationLifetimeActions[];
+}
+
+// @public
+export interface KeyRotationPolicyAttributes {
+    readonly created?: Date;
+    readonly expiryTime?: string;
+    readonly updated?: Date;
+}
+
+// @public
+export interface KeyRotationPolicyParameters {
```

Comment:
just FYI this is the top-level interface. eslint just doesn't like when things are referred to before they're defined so it reads bottom up instead of top-down

Reply:
Shouldn't it be an "Options"? Review certificates for the lifetime actions design there. We should mimic that as close as possible for consistency. Some flattening may be required.